cmd/secrets-store-csi-driver/main.go

```diff
@@ -52,15 +51,12 @@ var (
 	logFormatJSON      = flag.Bool("log-format-json", false, "set log formatter to json")
 	providerVolumePath = flag.String("provider-volume", "/etc/kubernetes/secrets-store-csi-providers", "Volume path for provider")
 	// this will be removed in a future release
-	_           = flag.String("min-provider-version", "", "[DEPRECATED] set minimum supported provider versions with current driver")
-	metricsAddr = flag.String("metrics-addr", ":8095", "The address the metric endpoint binds to")
-	// grpcSupportedProviders is a ; separated string that can contain a list of providers. The reason it's a string is to allow scenarios
-	// where the driver is being used with 2 providers, one which supports grpc and other using binary for provider.
-	grpcSupportedProviders = flag.String("grpc-supported-providers", "", "set list of providers that support grpc for driver-provider [alpha]")
```

Comment:
As we have done with other flags, should we mark it as deprecated and to be removed, but not actually remove until 2 versions from now?

Reply:
Ah sounds good, I'll leave the flag in. I did intend to remove _setting_ the flag in the yaml and helm chart, so the upgrade path would immediately stop setting it, but I can see value in having the flag just be ignored for a few releases in case anyone is manually updating just the image tag in their yaml.